components/kyma-environment-broker/internal/orchestration/manager/manager.go

```diff
@@ -60,6 +63,9 @@ func (m *orchestrationManager) Execute(orchestrationID string) (time.Duration, e
 		return m.failOrchestration(o, errors.Wrap(err, "while resolving operations"))
 	}
 
+	o.Parameters.Kyma.Version = m.cfg.KymaVersion
```

Comment:
This code should be moved to `resolveOperations()` in the branch which handles the `Pending` (to be started) orchestration. 
Teh reason is that if the orchestration is not pending (i.e. it is a resumed in progress after restart), it would overwrite parameters from config. Furthermore the parameters should only be set to the default config values, if they are empty, i.e. for each parameter it was not provided as input.